.clj-kondo/config.edn

```diff
@@ -672,7 +672,8 @@
   honey-sql-2-namespaces
   {:linters
    {:discouraged-namespace
-    {honeysql.core                     {:message "Use Honey SQL 2"}
+    {clj-yaml.core                     {:message "Use metabase.util.yaml"}
+     honeysql.core                     {:message "Use Honey SQL 2"}
```

Comment:
Should this be a top-level `:discouraged-namespace`? Instead of only for the `honey-sql-2-namespaces`